# A timestamped vector

std_msgs/Header header
float64[] values